src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -3880,12 +3880,12 @@ public void M()
             rest.ToString(); // 1
 
         if (new C<int?>() is [1, ..var rest2])
-            rest2.Value.ToString(); // 2
+            rest2.Value.ToString(); // 2 (assumed not-null)
```

Comment:
This isn't quite what I meant: the numbers are wrong, because warning 2 is no longer there.

Reply:
I renumbered the one you commented on. Will update the rest in the next commit.

Reply:
Done.